gslib/commands/signurl.py

```diff
@@ -162,6 +162,13 @@
                Note that both options have a maximum allowed duration of
                12 hours for a valid link.
 
+  -b <project> Allows you to specify a user project that will be billed for
+               requests on the signed URL. This is useful for generating
+               presigned links for buckets that use requester pays.
+
+               Note that it's not valid to specify both the ``-b`` and ``-u``
+               options together.
```

Comment:
Why wouldn't this be valid? Wouldn't the -u flag specify the account to bill for executing the `gsutil signurl` command, and the -b flag specify the account to bill when someone uses the signed URL that `gsutil signurl` generates?

Reply:
It's super annoying that there's a **local** `-u` and a **global** `-u` and I can VERY much understand how you could get those 2 confused. To your point, using both **global** `-u` and local `-b` _is_ a valid use case. I'm going to replace `-u` with `--use-service-account` to hopefully clear up the confusion.